c2corg_ui/static/js/documentediting.js

```diff
@@ -559,12 +548,24 @@ app.DocumentEditingController.prototype.preview = function() {
   var payload = {
     'document': angular.copy(this.scope[this.modelName])
   };
+  // TODO: loading indicator?
   this.http_.post(url, payload, config).
     catch(function(response) {
       this.alerts.addError('A server error prevented creating the preview');
     }.bind(this)).
     then(function(response) {
-      $('#preview-container-content').html(response['data']);
+      var template = angular.element('#preview-container').clone();
+      template.contents().filter(function() {
```

Comment:
You want to get the element with id `preview-container-content` inside the element `#preview-container`? Maybe you could use [find](https://api.jquery.com/find/) instead?